geode-core/src/distributedTest/java/org/apache/geode/cache30/GlobalRegionDUnitTest.java

```diff
@@ -240,80 +206,66 @@ public void uncaughtException(Thread t, Throwable e) {
         };
 
         Thread[] threads = new Thread[threadsPerVM];
-        for (int i = 0; i < threadsPerVM; i++) {
-          Thread thread = new Thread(group, new Runnable() {
-            @Override
-            public void run() {
+        for (int i1 = 0; i1 < threadsPerVM; i1++) {
+          Thread thread = new Thread(group, () -> {
```

Comment:
Optional: We should try to replace all testing `Threads` like these with use of `ExecutorServiceRule`. Spinning up our own `Thread` should be considered `deprecated` and replaced with `Executors`. The `ExecutorServiceRule` will report a failure if any thread is left running after the test or if any of these threads threw anything.